person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unkown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @parent_permission = parent_permission
+    @age = age
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission == true
+  end
```

Comment:
- [ ] As suggested by the previous reviewer, since by default `@parent_permission` is true there is no need to set it equal to true again in the `can_use_sevices?` method. Kindly remove the `== true` before you merge to avoid code repetition and follow the DRY rule.

Reply:
Thanks @ZeenatLawal